File: SimERP/ClientApp/src/app/datafilter/reportcolumnview/model/reportcolumnview.ts
import {Guid} from 'guid-typescript';

export class ReportColumnView {
  Id? = Guid;
  UserId? = -1;
  ColumnCode?: string;
  CreatedDate?: Date;
  CreatedBy?: number;
  ModifyDate?: Date;
  ModifyBy?: number;
  ViewType?: number;
  ColumnName?: string;

  // custom properties
  SortOrder: number;
}

File: SimERP.Data/DBEntities/ReportColumnView.cs
using System;
using System.Collections.Generic;

namespace SimERP.Data.DBEntities
{
    public partial class ReportColumnView
    {
        public string Id { get; set; }
        public int UserId { get; set; }
        public string ColumnCode { get; set; }
        public DateTimeOffset? CreatedDate { get; set; }
        public int? CreatedBy { get; set; }
        public DateTimeOffset? ModifyDate { get; set; }
        public int? ModifyBy { get; set; }
        public int ViewType { get; set; }
        public string ColumnName { get; set; }
    }
}
